robustredirects/models.py

```diff
@@ -62,3 +63,11 @@ class Meta:
     def __unicode__(self):
         return _("Redirect: %(from)s --> %(to)s") % {'from': self.from_url, 'to': self.to_url}
 
+
+    @staticmethod
+    def ignored_url_paths():
+        try:
```

Comment:
IMHO, this could be done more cleanly with
```suggestion
        url_paths = getattr(settings, "ROBUST_REDIRECTS_IGNORED_PREFIXES", ())
```